README.md

```diff
@@ -0,0 +1,130 @@
+# Tahoe-LAFS Infrastructure
+
+The software that defines the Tahoe-LAFS project's infrastructure.
+
+## Abstract
+
+This repository was initially created in Sep 2022 as an effort to deploy and
+manage a `testgrid` server using NixOS (now retired).
+
+Since January 2025, some members of the community are seeking to re-use this repository
+to deploy and manage some new systems:
+
+- a `testgrid` server re-using the NixOS configuration written in 2022
+- a `webforge` server to [MoveOffTrac](https://github.com/tahoe-lafs/MoveOffTrac)
+
+In addition to the [Nix](https://nixos.org/) code allowing to manage the software installed on those servers,
+an [OpenToFu]() plan should handle the definition of the related infrastructure (e.g.: VMs and DNS records).
+
+## ToDo/WiP
+
+The content of this repository is mostly a work in progress (as time of writing).
+
+Here follows a list of things to do, if not already in progress (see open issues).
```

Comment:
Why not have these in issues?  Seems weird to have to change `README.md` all the time, no?

Reply:
To make the README shorter, I've move this section in a separate document (docs/Roadmap.md).

But I really think it is rather important to state early in the README that the repository does not work yet as intended (hence the roadmap).